### PR TITLE
BREAKING new ilp-packet format

### PR DESCRIPTION
**Pull request by sentientwaffle**
Depends on https://github.com/interledgerjs/ilp-routing/pull/32

### CODE REVIEW

File: src/lib/client.js [hunk 1]
```diff
@@ -178,13 +175,12 @@ class Client extends EventEmitter {
...
 
-    const transferData = {
-      ilp_header: omitUndefined({
-        account: params.destinationAccount,
-        amount: params.destinationAmount,
-        data: params.destinationMemo
-      })
-    }
+    const ilpHeader = packet.serializeIlpPayment({
+      type: 'ilp',
```
Comment (emschwartz) [inline src/lib/client.js]: `type` isn't a parameter expected by the packet lib

Comment (sentientwaffle) [inline src/lib/client.js]: Fixed

File: src/lib/client.js [hunk 2]
```diff
@@ -178,13 +175,11 @@ class Client extends EventEmitter {
...
     }
 
-    const transferData = {
-      ilp_header: omitUndefined({
-        account: params.destinationAccount,
-        amount: params.destinationAmount,
-        data: params.destinationMemo
-      })
-    }
+    const ilpHeader = packet.serializeIlpPayment({
```
Comment (emschwartz) [inline src/lib/client.js]: Sorry to be a stickler about this but can you change this to be `ilpPayment`? "ILP Header" was the old name but we don't really have the concept of headers in the same way anymore

Comment (sentientwaffle) [inline src/lib/client.js]: Thanks; fixed

**Pull request merged**